config_example.json

```diff
@@ -28,7 +28,10 @@
             "devedition_candidates_repacks": "/path/to/beetmoverscript/beetmoverscript/templates/devedition_candidates_repacks.yml"
         },
         "push-to-releases": {},
-        "push-to-staging": {}
+        "push-to-partner": {
+            "firefox_partner": "/path/to/beetmoverscript/beetmoverscript/templates/firefox_partner.yml",
+            "firefox_partner_repacks": "/path/to/beetmoverscript/beetmoverscript/templates/firefox_partner_repacks.yml"
```

Comment:
I don't know this code, but I suspect that we don't need both of these. Partner repacks don't happen until after en-US + l10n is done, and all locales (including en-US) are available at the same time.

Reply:
Hm, possible, especially if we create the payload so `en-US` is an explicit locale, rather than assumed.